mirascope/core/cohere/call_response.py

```diff
@@ -1,5 +1,6 @@
 """This module contains the `CohereCallResponse` class."""
 
+from functools import cached_property
```

Comment:
what's this import for?

Reply:
originally fixed a bug but later realized the bug was somewhere else, removed.